policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/phase_out/increment.yaml

```
@@ -16,7 +17,9 @@ metadata:
   reference:
   # Legal code specifies $5,000 starting in 2011.
   - title: R.I. Gen. Laws § 44-30-2.6 (B), (III)
-    href: http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-30/44-30-2.6.HTM
+    href: https://law.justia.com/codes/rhode-island/title-44/chapter-44-30/part-i/section-44-30-2-6/
+  - title: RHODE ISLAND TAX RATE SCHEDULE AND WORKSHEETS 2024
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2024-12/2022%20Tax%20Rate%20and%20Worksheets.pdf
```

```suggestion
    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2024-12/2024%20Tax%20Rate%20and%20Worksheets.pdf
``` 
And other files that use this link